lib/statsd.js

```diff
@@ -1,6 +1,14 @@
 'use strict';
 var StatsD = require('hot-shots');
 
+/**
+ * Maximum size of a metrics batch used by default.
+ *
+ * @const
+ * @type {number}
+ */
+var DEFAULT_MAX_BATCH_SIZE = 1450;
```

Comment:
hehe.. Actually `var` in ES6 is considered legacy as there's not real use for it. So, when we drop node 0.10, which, I hope, will happen soon, we would probably replace `var` with `const` and `let` everywhere..